Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -873,10 +884,15 @@ extension MacroApplication {
   /// node.
   ///
   /// - Returns: The storage node with all macro-synthesized accessors applied.
```

Comment:
Could you add a sentence for `expandsGetSet`?

Reply:
Sure, the entire comment is wrong so I'll update it 👍 (it doesn't return the storage)